spec/index.bs

```diff
@@ -3464,7 +3464,7 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                                 - Let |storageTextureLayout| be |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}
 
                                 - Exactly one of |bufferLayout|, |samplerLayout|, |textureLayout|,
-                                    or |storageTextureLayout| are not `undefined`.
+                                    or |storageTextureLayout| [=map/exists=].
```

Comment:
One question I do have is if we feel it's appropriate to use `exists` in cases like this where we're referencing a variable that points to the dictionary value rather than the dictionary itself.

Reply:
In this case we are violating [this assertion](https://infra.spec.whatwg.org/#map-get) by accessing members of the dictionary that may not exist in the first place. I think we should just reword this without the local variables, it'll be only slightly more verbose.

Reply:
Okay, done. You're right that it didn't add that much verbosity to the text.